modular_skyrat/modules/liquids/code/liquid_systems/liquid_effect.dm

```diff
@@ -116,6 +116,9 @@
 	. = ..()
 	if(fire_state)
 		set_fire_state(LIQUID_FIRE_STATE_NONE)
```

Comment:
wouldnt it be better to add this new line to this proc itself?

Reply:
Upon further examination, it's right how it is.